tests/bin/agent/start.test.ts

```diff
@@ -569,4 +570,247 @@ describe('start', () => {
     },
     global.defaultTimeout * 2,
   );
+  describe('seed nodes', () => {
```

Comment:
This is a legitimate use of the nested describe to use the global seed node. It's also correctly at the bottom of the test.